src/test/java/mil/dds/anet/test/resources/AnetEmailResourceTest.java

```diff
@@ -0,0 +1,59 @@
+package mil.dds.anet.test.resources;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import mil.dds.anet.AnetObjectEngine;
+import mil.dds.anet.config.AnetConfiguration;
+import mil.dds.anet.test.TestData;
+import mil.dds.anet.test.client.AnetBeanList_Report;
+import mil.dds.anet.test.client.ReportSearchQueryInput;
+import mil.dds.anet.threads.AnetEmailWorker;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class AnetEmailResourceTest extends AbstractResourceTest {
+  @Autowired
+  protected DropwizardAppExtension<AnetConfiguration> dropwizardApp;
+  private AnetEmailWorker emailWorker;
+
+  @BeforeAll
+  void setUpClass() {
+    final AnetObjectEngine engine = AnetObjectEngine.getInstance();
+    emailWorker = new AnetEmailWorker(dropwizardApp.getConfiguration(), engine.getEmailDao());
+    emailWorker.run();
+  }
+
+  @AfterAll
+  void tearDownClass() {
+    emailWorker.run();
+    emailWorker = null;
+    AnetEmailWorker.setInstance(null);
+  }
+
+  @Test
+  void shouldReturnEmailData() {
+    final var reportQuery = ReportSearchQueryInput.builder()
+        .withAuthorPositionUuid(admin.getPosition().getUuid()).build();
+
+    // Search by author position
+    final AnetBeanList_Report results = withCredentials(adminUser,
+        t -> queryExecutor.reportList(getListFields("{ uuid }"), reportQuery));
+    final var reportUuid = results.getList().iterator().next().getUuid();
+    final var input = TestData.createAnetEmailInput();
+    final var mutationResult =
+        withCredentials(adminUser, t -> mutationExecutor.emailReport("", input, reportUuid));
+    assertThat(mutationResult).isEqualTo(1);
+
+    final var pendingEmails = withCredentials(adminUser, t -> queryExecutor.pendingEmails(""));
```

Comment:
You're not retrieving any fields in your query!

Reply:
The assertion compares the retrieved email against the input (minus the fields that are assigned on persisting):

`    assertThat(pendingEmails).isNotEmpty()
        .anySatisfy(pendingEmail -> assertThat(pendingEmail).usingRecursiveComparison()
            .ignoringFields("__typename", "id", "createdAt").isEqualTo(input));`

Reply:
You should explicitly list the fields you want to retrieve in the query response, not use `""` (which is non-standard, and you only get away with it because the library is forgiving, and gets all fields incl. the private ones like `__type`). Look at other queries in the tests for some examples.